{"kind":"symbol","primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"system","kind":"identifier"}],"languages":["swift"],"platforms":["macOS"]}]}],"identifier":{"url":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute\/system","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit","doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute"]]},"metadata":{"roleHeading":"Case","title":"SourceErrorAttribute.system","role":"symbol","externalID":"s:13SwiftErrorKit06SourceB9AttributeO6systemyA2CmF","symbolKind":"case","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"system"}],"modules":[{"name":"SwiftErrorKit"}]},"sections":[],"variants":[{"paths":["\/documentation\/swifterrorkit\/sourceerrorattribute\/system"],"traits":[{"interfaceLanguage":"swift"}]}],"abstract":[{"text":"Error caused by a system that the process runs on.","type":"text"},{"type":"text","text":" "},{"type":"text","text":"Hardware failure, OS-level error, resource exhaustion,"},{"text":" ","type":"text"},{"text":"environment configuration, …","type":"text"}],"schemaVersion":{"minor":3,"major":0,"patch":0},"references":{"doc://SwiftErrorKit/documentation/SwiftErrorKit/SourceErrorAttribute":{"url":"\/documentation\/swifterrorkit\/sourceerrorattribute","role":"symbol","navigatorTitle":[{"text":"SourceErrorAttribute","kind":"identifier"}],"type":"topic","abstract":[{"type":"text","text":"An error attribute to include information about the high-level component"},{"text":" ","type":"text"},{"text":"in which the error emerged.","type":"text"}],"fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"SourceErrorAttribute","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute","title":"SourceErrorAttribute"},"doc://SwiftErrorKit/documentation/SwiftErrorKit":{"type":"topic","role":"collection","abstract":[{"type":"text","text":"Error attribution and commons."}],"url":"\/documentation\/swifterrorkit","identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit","kind":"symbol","title":"SwiftErrorKit"},"doc://SwiftErrorKit/documentation/SwiftErrorKit/SourceErrorAttribute/system":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"system"}],"identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute\/system","role":"symbol","url":"\/documentation\/swifterrorkit\/sourceerrorattribute\/system","type":"topic","kind":"symbol","title":"SourceErrorAttribute.system","abstract":[{"text":"Error caused by a system that the process runs on.","type":"text"},{"text":" ","type":"text"},{"text":"Hardware failure, OS-level error, resource exhaustion,","type":"text"},{"text":" ","type":"text"},{"text":"environment configuration, …","type":"text"}]}}}